### ISSUE TITLE
Broken dependency: libgconf-2

### ISSUE THREAD

**Issue by joelgraff** [resin-io/etcher#2316]
Downloaded the latest AppImage and ran it on a clean install of Pop!_OS (Bionic).  Reported a missing dependency.  Installed libgconf-2-4 and it ran without issues.

**Comment by jhermsmeier** [comment]
Duplicate of #1305

**Comment by probonopd** [comment]
Unbelievable it's still not fixed in 1.4.9. It's been almost a year now...

```
me@host:~$ '/home/me/Downloads/balena-etcher-electron-1.4.9-x86_64.AppImage'
2993
/tmp/.mount_G2Gbv3/usr/bin/balena-etcher-electron: error while loading shared libraries: libgconf-2.so.4: cannot open shared object file: No such file or directory
```
